### ISSUE TITLE
Multiple issues when mass-running analyzers (reloading the page kills analyzers and more)

### ISSUE THREAD

**Issue by github-pba** [TheHive-Project/TheHive#1095]
### Request Type
Bug

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Ubuntu
| OS version (client)       |  Win7
| TheHive version / git hash   | 3.4.0-RC2
| Package Type              | DEB
| Browser type & version    | Chrome 73.0.3683.75, IE11
| Elastic4Play | 1.11.4
| Play | 2.6.23
| Elastic4s | 6.5.1
| ElasticSearch | 6.5.2

### Problem Description
When running an analyzer for a huge number of observables,
* the UI gets unresponsive
* can't switch to admin functions
* reloading the UI terminates analyzer jobs

Users on other browsers are not affected and can still work.

### Steps to Reproduce
1. create a case with many IP observables (I fed 355 IPs from [feodotracker](https://feodotracker.abuse.ch/downloads/ipblocklist.csv))
2. choose all observables and run the GoogleDNS_Resolve analyzer.

This will generate 355 background jobs.

3. Click on TheHive icon to go to the list of cases.
None will be displayed. It can't even determine the number of open cases.
![ss002](https://user-images.githubusercontent.com/50981940/62940774-914d7600-bdd4-11e9-861a-fcda70737a48.png)

4. Click on Admin -> Case Templates
The page reloads, and you will only see the spinner (forever? As long as the analyzers run? Didn't test)

5. Press F5 to reload the page
The page is reloaded, but most of the background analyzer jobs vanish and won't get executed. This can be seen when looking at the reports for each observable.

6. select all the observables once again and restart the GoogleDNS_Resolve analyzer. When the background jobs are running, hit F5 to reload the page. 
The page is reloaded, but most of the background analyzer jobs vanish and will not get executed.

7. Restart the mass-analyzing again and wait patiently.
The analyzers will successfully finish.

**Conclusion:**
When mass-analyzing observables, you may not reload the page and can't use the current browser as long analysis runs.

Yoda: no reload, you shall do

**Comment by crackytsi** [comment]
As a lot of things are going to happen and initiated on browser level: Are you sure you have enough memory? I run into this issue doing actions to a lot of things...

**Comment by github-pba** [comment]
Yes, I'm quite sure my brower has enough memory. My PC has 8GB, where 4GB is free when I run my common business setup.

**Comment by shubb30** [comment]
I can confirm that this is due to the browser refresh.  Once you submit analyzers, you have to wait for the jobs counter to get to zero for them to be all submitted.

Quite honestly, it blew my mind that this is the mechanism the hive is using to submit analyzer jobs...

**Comment by pinowudi** [comment]
since the number of analyzers to run can be a multiple of the number of observables, and many cases have tens to hundreds of these... It can be a bit bogged on the local javascript to submit all of these. Errors and non-submits abound, especially for analysts further away from theHive. Distance kills performance.
